README.md

```diff
@@ -1,14 +1,34 @@
 # YouTube-homepage
-Building an HTML document that matches the appearance of YouTube .
+Building an HTML and css page that matches the appearance of this youtube page https://www.youtube.com/watch?v=f9ip1iSAptQ&t=2sYouTube
 # Built With
-- HTML
-- CSS
+- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML),
+- [CSS](https://www.w3schools.com/css/),
+- [Fontawesome](https://fontawesome.com/)
+
+# Original Page
+<p align="center">
+  <img src="img/project-screenshot.png" width="850" title="YouTube screenshot">
+</p>
```

Comment:
### OPTIONAL
This section ideally should be a screenshot of your clone to showcase what you have done, not a screenshot of the original page. In your future projects, please use a screenshot of your project.